pkg/log/log_messages.go

```diff
@@ -80,3 +80,17 @@ const CAKC064 string = "CAKC064 Unable to find Authenticator for this configurat
 const CAKC065 string = "CAKC065 File path is invalid %s"
```

Comment:
when will this log appear?

Reply:
When use will put file path like this : "a/////b" which is 100% invalid path